example-theme/theme.edn

```diff
@@ -0,0 +1,24 @@
+{:color1 "#aa0000"
+ :color2 "#00aa00"
+ :color3 "#0000aa"
+ :color4 "#aa00aa"
+ ;; these are just the defaults:
```

Comment:
The defaults don't need to be repeated here, because they will be inherited from config-defaults.edn

Reply:
yes, but now it's a better example (for docs)

Reply:
👌